<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionDetails</class>
 <widget class="QConnectedTabWidget" name="SelectionDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selection Details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnSelectAll">
         <property name="text">
          <string>Check All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUnselectAll">
         <property name="text">
          <string>Uncheck All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSelectTagged">
         <property name="text">
          <string>Check Tagged</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUnselectTagged">
         <property name="text">
          <string>Uncheck Tagged</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnShowAllSelected">
         <property name="text">
          <string>Show All Selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnShowTaggedSelected">
         <property name="text">
          <string>Show Tagged Selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnShowTaggedAll">
         <property name="text">
          <string>Show All Tagged</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeView" name="listSelection">
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="btnShowSummary">
             <property name="text">
              <string>Source Histograms</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnShowTrend">
             <property name="text">
              <string>Show Trend</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnTagSelected">
             <property name="text">
              <string>Tag Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbTagList"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddTag">
             <property name="text">
              <string>Add Tag</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QConnectedTabWidget</class>
   <extends>QWidget</extends>
   <header>QConnectedTabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
